index.html

```diff
@@ -1719,51 +1721,25 @@ <h2>Capability Delegation</h2>
       </section>
 
       <section>
-        <h2>Resource Integrity</h2>
-
-        <p class="issue atrisk" title="Unification of cryptographic hash expression formats are under discussion">
-The Working Group is currently attempting to determine whether cryptographic hash
-expression formats can be unified across all of the VCWG core specifications.
-Candidates for this mechanism include `digestSRI` and `digestMultibase`. There
-are arguments for and against unification that the WG is currently debating.
-        </p>
-
-        <p>
-When including a link to an external resource in a
-<a>conforming secured document</a>, it is desirable to know whether the resource
-that is identified has changed since the proof was created. This applies to
-cases where there is an external resource that is remotely retrieved as well as
-to cases where the <a>verifier</a> might have a locally cached copy of the
-resource.
-        </p>
-        <p>
-To enable confirmation that a resource referenced by a <a>conforming secured document</a>
-has not changed since the document was secured, an implementer MAY include a
-property named <dfn class="lint-ignore">`digestMultibase`</dfn> in any object
-that includes an `id` property. If present, the `digestMultibase` value MUST be
-a single string value, or an array of string values that are
-[[MULTIBASE]]-encoded [[MULTIHASH]] values.
-        </p>
+        <h2>Multibase</h2>
 
-        <p>
-An example of a resource integrity protected object is shown below:
+        <p class="issue atrisk"
+           title="Multibase and Multihash may be standardized at IETF">
+The [[?MULTIBASE]] specification has been dispatched at IETF and may be
+standardized there. There is active discussion on this initiative in
+the <a href="https://mailarchive.ietf.org/arch/browse/multiformats/">
+Multiformats mailing list at IETF</a>. If the Multibase draft is stabilized
+before this specification goes to the Proposed Recommendation phase, the table
+below will be replaced with normative references to the Multibase
+specification at IETF. It is the intention of the Working Group to ensure
+alignment between the Multibase values used in this specification and the
+Multibase values defined by the current Multibase community and any potential
+future IETF Multiformats Working Group.
         </p>
 
-        <pre class="example nohighlight"
-          title="An integrity-protected image that is associated with an object">
-{
-  ...
-  "image": {
-    "id": "https://university.example.org/images/58473",
-    "digestMultibase": "zQmdfTbBqBPQ7VNxZEYEj14VmRuZBkqFbiwReogJgS1zR1n"
-  },
-  ...
-}
-        </pre>
-
         <p>
-Common [[MULTIBASE]] header values and their encoding alphabets are provided
-below:
+The common Multibase header values and their encoding alphabets
+as provided below are normative:
```

Comment:
```suggestion
A Multibase string includes a single character header which identifies the base
and encoding alphabet used to encode a binary value, followed the encoded
binary value (using that base and alphabet). The common Multibase header
values and their associated base encoding alphabets as provided below are
normative:
```

Reply:
Thanks, @dlongley, this is the kind of specification text I was looking for.

Although not part of the original PR, the same kind of text should also be added to the section on Multihash (which, in its current form, is no less cryptic)